GBF/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "author": {
+        "name": "Morrays",
+        "id": ""
+    },
+    "service": "GBF",
+    "description": {
+        "en": " Japanese role playing game"
+        
+    },
+    "url": "game.granbluefantasy.jp",
+    "version": "1.0",
+    "largeImageKey": "logo",
+    "logo": "https://i.imgur.com/W0nvASk.png",
+    "thumbnail": "https://i.imgur.com/W0nvASk.png",
+    "color": "#FFFFFF",
```

Comment:
can you change the thumbnail to a screenshot of the website or the game itself and the color as well